docs/mesos/RpmInstallMesos.md

```diff
@@ -0,0 +1,51 @@
+# Create and Install the Mesos + Net-Modules RPM
+This guide will walk you through building and installing Mesos + Net-Modules from RPMs. These RPMs will automate the installation of everything needed to run Mesos + Net-Modules in your cluster.
+
+## Build and Install the RPM
```

Comment:
I would like this to be download and install, rather than build and install.  What do we need to get there?  (This one is even more important that the plugin, since Mesos is a bear to build.)